tcms/templates/case/clone.html

```diff
@@ -20,53 +21,53 @@
 		<input id="value_plan_id" type="hidden" name="plan_id" value="{{ test_plan.plan_id }}" />
 	</div>
 
-	<h2>Clone TestCase(s)</h2>
+	<h2>{% trans "Clone TestCase(s)" %}</h2>
 	<div class="Detailform border-1">
 		<div class="editor" cellspacing="0" cellspan="0">
 			<div class="listinfo">
-				<label class="strong title">Select Plan </label>
+				<label class="strong title">{% trans "Select Plan " %}</label>

```

Comment:
You don't need the trailing space in source string.

Reply:
Ok